[2025-02-24T04:55:08.846+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:55:08.866+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.get_name scheduled__2025-02-23T00:00:00+00:00 [queued]>
[2025-02-24T04:55:08.877+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.get_name scheduled__2025-02-23T00:00:00+00:00 [queued]>
[2025-02-24T04:55:08.878+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 7
[2025-02-24T04:55:08.890+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): get_name> on 2025-02-23 00:00:00+00:00
[2025-02-24T04:55:08.897+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1497) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:55:08.897+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'get_name', 'scheduled__2025-02-23T00:00:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpo6v0955r']
[2025-02-24T04:55:08.899+0000] {standard_task_runner.py:72} INFO - Started process 1502 to run task
[2025-02-24T04:55:08.899+0000] {standard_task_runner.py:105} INFO - Job 121: Subtask get_name
[2025-02-24T04:55:08.940+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.get_name scheduled__2025-02-23T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:55:09.011+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T00:00:00+00:00'
[2025-02-24T04:55:09.012+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:55:09.040+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-24T04:55:09.047+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:55:09.048+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=get_name, run_id=scheduled__2025-02-23T00:00:00+00:00, execution_date=20250223T000000, start_date=20250224T045508, end_date=20250224T045509
[2025-02-24T04:55:09.073+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:55:09.091+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:55:09.093+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
